<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Faster Than Sight</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CReaderViewWidget" name="_text">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent">
     <property name="title">
      <string>Open &amp;recent</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Reading">
    <property name="title">
     <string>&amp;Reading</string>
    </property>
    <addaction name="action_Read"/>
    <addaction name="action_Pause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious_chapter"/>
    <addaction name="actionPrevious_paragraph"/>
    <addaction name="actionNext_paragraph"/>
    <addaction name="actionNext_chapter"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_word"/>
   </widget>
   <widget class="QMenu" name="menu_Bookmarks">
    <property name="title">
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Bookmark_current_position"/>
    <addaction name="action_Remove_bookmarks"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Fullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionShow_pivot"/>
    <addaction name="actionClear_screen_after_sentence_end"/>
    <addaction name="separator"/>
    <addaction name="action_Themes"/>
    <addaction name="action_Font"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuAdvanced">
     <property name="title">
      <string>Advanced</string>
     </property>
     <addaction name="actionView_application_log"/>
    </widget>
    <addaction name="actionView_navigate_text"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="menuAdvanced"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Bookmarks"/>
   <addaction name="menu_Reading"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Read"/>
   <addaction name="action_Pause"/>
   <addaction name="actionPrevious_chapter"/>
   <addaction name="actionPrevious_paragraph"/>
   <addaction name="actionNext_paragraph"/>
   <addaction name="actionNext_chapter"/>
   <addaction name="actionPrevious_word"/>
   <addaction name="actionNext_word"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="text">
    <string>&amp;Font...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="action_Read">
   <property name="text">
    <string>&amp;Read</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop and reset to the beginning</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionShow_pivot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;pivot</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionGo_to_word">
   <property name="text">
    <string>&amp;Go to word...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Themes">
   <property name="text">
    <string>Themes...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="action_Bookmark_current_position">
   <property name="text">
    <string>&amp;Bookmark current position</string>
   </property>
  </action>
  <action name="action_Remove_bookmarks">
   <property name="text">
    <string>&amp;Remove bookmarks...</string>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="text">
    <string>&amp;Fullscreen (F11)</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionClear_screen_after_sentence_end">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Clear screen after sentence end</string>
   </property>
  </action>
  <action name="actionPrevious_chapter">
   <property name="text">
    <string>Previous chapter</string>
   </property>
  </action>
  <action name="actionPrevious_paragraph">
   <property name="text">
    <string>Previous paragraph</string>
   </property>
  </action>
  <action name="actionNext_paragraph">
   <property name="text">
    <string>Next paragraph</string>
   </property>
  </action>
  <action name="actionNext_chapter">
   <property name="text">
    <string>Next chapter</string>
   </property>
  </action>
  <action name="actionView_navigate_text">
   <property name="text">
    <string>View / &amp;navigate text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for &amp;updates</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionView_application_log">
   <property name="text">
    <string>View application log...</string>
   </property>
  </action>
  <action name="actionPrevious_word">
   <property name="text">
    <string>Previous word</string>
   </property>
   <property name="toolTip">
    <string>Previous word</string>
   </property>
  </action>
  <action name="actionNext_word">
   <property name="text">
    <string>Next word</string>
   </property>
   <property name="toolTip">
    <string>Next word</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>CReaderViewWidget</class>
   <extends>QQuickWidget</extends>
   <header>QML/creaderviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
